docs/examples/pcrglobwb.ipynb

```diff
@@ -2,57 +2,130 @@
  "cells": [
```

Comment:
<p>I found this kind of assignments <code>custom_parameter_set = parameter_set =</code> confusing and unnecessary. Can we just use <code>parameter_set</code> ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eWaterCycle/ewatercycle/pull/167/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
oops, that was a mistake. I intended to only write `custom_parameter_set = ParameterSet(...)`. I'd like it to keep the 'custom' in the name, though.